pymodbus/framer/base.py

```diff
@@ -27,14 +27,14 @@ class ModbusFramer:
     def __init__(
         self,
         decoder: ClientDecoder | ServerDecoder,
-        client=None,
+        **kwargs
```

Comment:
client= is a named parameter, why make the anonymous....also the class do not allow other parameters.

Reply:
same here